lib/babel-schmooze-sprockets/engine.rb

```diff
@@ -1,7 +1,4 @@
 module BabelSchmoozeSprockets
   class Railtie < Rails::Engine
-    initializer "babel-schmooze-sprockets" do
-      config.assets.paths << File.expand_path("#{__dir__}/../../node_modules")
-    end
   end
 end
```

Comment:
Actually, it's not a Railtie, but an Engine. And it adds tons of things to the load path, including what @lucasmazza mentioned. It also allows defining custom routes, migrations, and more.

I fixed the naming issue on 51701ddc5a17a418cab2178fa3267ea19b34a79b.